docs/references/beta-only-features.md

```diff
@@ -79,8 +79,15 @@ feature.
 
 ## Stabilizing a feature
 
-    TODO: How to move a feature from beta-only to building unconditionally.
+Careful scoping of [defined features](../../common/config/babel/.babelrc.js) will allow you to easily stabilize the feature. When you are ready to add your feature to the stable build:
 
+- One-step stabilization: Include all associated conditionally compiled code in the stable flavored build by moving your feature to `stabilizedFeature` in the [defined features config file]((../../common/config/babel/.babelrc.js)).
+  - Switch to stable flavor and build, test and run samples to try out your stabilized feature.
+  - You will also get the updated API files for the stable build from this step. You can use this for API review.
+- Clean up: Once your feature is shipped in a stable release, remove all references to conditional compilation directive for your feature, and remove your feature from `stabilizedFeature` list.
```

Comment:
Why do this separately after the build has been shipped?
If this is done originally, there would be no need for adding it to stabilizedFeature

If this needs to be done anyway, I don't see the effectiveness of stabilizedFeature :P

Reply:
Also if `stabilizedFeature` is a list, shouldn't it be `stabilizedFeatures` :D

Reply:
The value of `stabilizedFeatures` is that it gives us a single flag flip to stabilize (and make beta-only if need be) a feature.

The cleanup will take longer, can be done in smaller chunks, and would not risk destabilizing a release.
When we clean up, it will be more than just removing the conditional compilation directives. Various `*trampoline` functions we've created should be deleted as they are no longer needed etc.

This separates the (simple, but release critical) act of releasing a feature from (more complex, but no longer on the hot-path) cleaning up the conditional compilation related logic.